modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/search_container.js

```diff
@@ -119,6 +119,11 @@ AUI.add(
 
 							row.removeClass(CSS_TEMPLATE);
 
+							row.attr(
+								'id',
+								instance._originalConfig.id + '_' + id
```

Comment:
Well, it's a complete mystery to me where `_originalConfig` comes from... `git grep` gives four hits in liferay-portal (three under `modules/apps/archived` and a fourth in the inauspiciously name `js/legacy/upload.js`). Is this some kind of magical AUI thing, is it?

Assuming it is, this change looks fine to me.

Reply:
love you @wincent

Reply:
It is indeed a magical AUI thing -> https://github.com/liferay/alloy-ui/blob/master/src/aui-component/js/aui-component.js#L45

![neverendingstory](https://user-images.githubusercontent.com/905006/117288255-38085e80-ae63-11eb-969b-8f9131635c1b.gif)

Reply:
What I don't understand is why we can't just use `instance.get('id')`, since I assume `instance` here is still `SearchContainer` and `config.id === originalConfig.id` 🤷

Reply:
There is no reason, truthfully.